lib/ansible/modules/user.py

```diff
@@ -1218,6 +1218,9 @@ def modify_user(self):
 
     def create_homedir(self, path):
         if not os.path.exists(path):
+            if path == "/dev/null":
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example. Also, write a test for this change [here](https://github.com/ansible/ansible/tree/devel/test/integration/targets/user/tasks).

Reply:
this is changing detection on home dir not on skeleton (see a few lines below) also you will need to do this for more than just this version of create_homedir.